container/functional/keylime_agent_container-basic-attestation/main.fmf

```diff
@@ -0,0 +1,29 @@
+summary: Tests basic keylime attestation scenario for keylime agent container
+description: |
+    Running only verifier and registrar on localhost. Agents run in containers.
+    Uses certificates generated by keylime.
+    Starts verifier, registrar on localhost.
+    Build container image with agent. 
+    Setup agent conf for each container.
+    Run container with agent.
+    Resgister agent by verifier.
+    Verifiers that container with agent passed attestation.
+    Does changes on a system and verifies that one container system has failed attestation. Second container again passed attestation.
+contact: Patrik Koncity <pkoncity@redhat.com>
+component:
+  - keylime
+test: ./test.sh
+framework: beakerlib
+tag:
+  - CI-Tier-1
+require:
+  - yum
+  - podman
+recommend:
+  - keylime
+duration: 5m
```

Comment:
Maybe 5m won't be enough given we need to download the image. I would maybe go with 10m.